docs/product/issues/issue-details/breadcrumbs/index.mdx

```diff
@@ -1,61 +1,45 @@
 ---
 title: Using Breadcrumbs
-description: "Learn about using breadcrumbs on the Issue Details page to help resolve issues."
+description: "Learn about using breadcrumbs on the Issue Details page to debug."
 sidebar_order: 40
 ---
 
 Sentry uses _breadcrumbs_ to create a trail of events that happened prior to an issue. These events are very similar to traditional logs, but can record more rich structured data.
 
 SDKs automatically start recording breadcrumbs by enabling integrations. To learn how to do this for your platform, select the platform from the dropdown at the top of the page, then review the breadcrumbs documentation for that platform.
 
-If an error event provides a trail, [sentry.io](https://sentry.io) displays the breadcrumbs on the **Issue Details** page:
+If an error event provides a trail, [sentry.io](https://sentry.io) displays the breadcrumbs on the **Issue Details** page.
 
-![Breadcrumbs section](./img/breadcrumbs-section.png)
+![Breadcrumbs section](../img/issue-breadcrumbs.png)
 
 ## Breadcrumbs Display
 
-Each part of the breadcrumb displays in a separate row and each row includes the following details: Type, Category, Description, Level, and Time.
+On the main issue details page, you'll only see a portion of the breadcrumbs that may be available for the event. To access the rest of them, click the 'View All' button at the bottom of the section. This will open a slide out drawer where you can search, filter, sort and change the time display for all the breadcrumbs. To close this drawer, click 'Close' or press the escape key.
```

Comment:
```suggestion
You'll see a few of the breadcrumbs that may be available for the event in the issue "Details" tab. To access the rest of them, click the "View All" button at the bottom of the section. This will open a slide out drawer where you can search, filter, sort, and change the time display for all available breadcrumbs. 
```
I don't think we need to add instructions for closing the section since it's pretty self-explanatory :)